engine/flink/management/periodic/src/test/scala/pl/touk/nussknacker/engine/management/periodic/DeploymentManagerStub.scala

```diff
@@ -47,7 +80,13 @@ class DeploymentManagerStub extends BaseDeploymentManager with StubbingCommands
   override def getProcessStates(
       name: ProcessName
   )(implicit freshnessPolicy: DataFreshnessPolicy): Future[WithDataFreshnessStatus[List[StatusDetails]]] = {
-    Future.successful(WithDataFreshnessStatus.fresh(jobStatus.toList))
+    Future.successful(WithDataFreshnessStatus.fresh(jobStatus.get(name).toList.flatten))
+  }
+
+  override def getProcessesStates()(
```

Comment:
I would call it `getAllProcessesStates()` to emphasize that it could be expensive